### PR TITLE
Create notifications page

### PR DESCRIPTION
**Pull request by Skczu**

### CODE REVIEW

File: validators/Notification.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  id: z.number(),
+  title: z.string(),
+  communityName: z.string(),
+  notificationType: z.enum([
+    'applicationAccepted',
+    'newApplication',
+    'newProject',
+    'projectVotingComplete',
+  ]),
+});
```
Comment (wisnie) [inline validators/Notification.ts:13]: No description?

File: components/composited/Notification/NotificationItem.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  communityName: string;
+  isNew?: boolean;
+};
+
+export const NotificationItem = ({
+  notificationType,
+  communityName,
+  isNew = false,
+}: NotificationItemProps) => {
+  const [name, descPart1, descPart2] = getNotificationContents(notificationType);
```
Comment (wisnie) [inline components/composited/Notification/NotificationItem.tsx]: Create function `getTitle`, and `getDescription` parts. I would probably do it in a different way, but we don't have much time to think.

File: components/composited/Notification/NotificationItem.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          {name}
+        </span>
+        {isNew && <Badge textContent="Nowe" color="amber" />}
+      </div>
+      <div>
+        <Text className="sm:text-sm" as="span" content={descPart1} />
+        <Link href={`/communities/${communityName}`}>
+          <LinkButton textContent={communityName} />
+        </Link>
+        <Text className="sm:text-sm" as="span" content={descPart2} />
```
Comment (wisnie) [inline components/composited/Notification/NotificationItem.tsx:39]: No, it's one text, with link embedded, so you use one `Text` and you wrap `Link` inside of it.

File: components/composited/Notification/getNotificationContents.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+export const getNotificationContents = (notificationName: NotificationType) => {
+  switch (notificationName) {
+    case 'applicationAccepted':
+      return ['Akceptacja podania', 'Twoje podanie do grupy ', ' zostało pomyślnie rozpatrzone.'];
+    case 'newApplication':
+      return ['Nowe podanie', 'Pojawiło się nowe podanie do twojej grupy ', '.'];
+    case 'newProject':
+      return ['Nowy projekt', 'Utworzono nowy projekt zagospodarowania budżetu w grupie ', '.'];
+    case 'projectVotingComplete':
+      return ['Zakończenie głosowania', 'Projekt w grupie ', ' oczekuje rozpatrzenia.'];
```
Comment (wisnie) [inline components/composited/Notification/getNotificationContents.ts]: Maybe add an enum/constant with those values.

File: components/composited/SignInForm/SignInForm.tsx [hunk 5]
```diff
@@ -47,7 +50,18 @@ export const SignInForm = ({ onSubmit }: SignInFormProps) => {
...
+            type="submit"
+            content="Zaloguj się"
+            variant="primary"
+          />
+          <Text className="font-bold" as="span" content="lub" />
+          <Link href="/signup">
+            <LinkButton textContent="Utwórz konto" />
+          </Link>
+        </div>
       </form>
```
Comment (wisnie) [inline components/composited/SignInForm/SignInForm.tsx]: I will check spacing later.

Comment (wisnie) [reply components/composited/SignInForm/SignInForm.tsx]: You can remind me.

File: components/composited/Notification/NotificationItem.tsx (hunk 3 above)
Comment (Skczu) [reply components/composited/Notification/NotificationItem.tsx:39]: I don't get your idea here. Could you elaborate?

File: validators/Notification.ts (hunk 1 above)
Comment (Skczu) [reply validators/Notification.ts:13]: Descriptions are always the same, we shouldn't store them.

**Comment by wisnie** [comment]
We will not have enough time to finish this.

**Pull request merged**